features/kinderen.feature

```diff
@@ -5,78 +5,121 @@ Functionaliteit: Kinderen van een persoon raadplegen
       | type                | RaadpleegMetBurgerservicenummer |
-      | burgerservicenummer | 999996150                       |
+      | burgerservicenummer | 999996381                       |
       | fields              | kinderen.naam.voornamen         |
-      Dan heeft de persoon met burgerservicenummer '999996381' een kind met de volgende 'naam' gegevens
+      Dan heeft de response een persoon met een 'kind' met de volgende 'naam' gegevens
       | naam      | waarde |
       | voornamen | Vica   |
-      En heeft de persoon met burgerservicenummer '999996381' een kind met de volgende 'naam' gegevens
+      En heeft de persoon een 'kind' met de volgende 'naam' gegevens
       | naam      | waarde |
       | voornamen | Bella  |
-      En heeft de persoon met burgerservicenummer '999996381' GEEN kind met de volgende 'naam' gegevens
-      | naam      | waarde          |
-      | voornamen | Celeke Lodivica |
 
-  @gba
   Rule: Een kind wordt alleen teruggegeven als minimaal één gegeven in de identificatienummers (groep 01), naam (groep 02) of geboorte (groep 03) van het kind een waarde heeft.
     - Wanneer in een categorie kind alleen gegevens zijn opgenomen in groep 81 of 82, 85 en 86, wordt dit kind niet opgenomen in het antwoord
     - Wanneer een gegeven een standaardwaarde heeft, zoals "." (punt) bij geslachtsnaam of "00000000" bij geboortedatum, geldt dat hier als het bestaan van een waarde en wordt het kind wel geleverd
     - Wanneer door de gebruikte fields parameter in het request het kind in de response geen enkel gegeven heeft met een waarde, dan wordt het kind geleverd zonder gegevens (dus als leeg object)
 
     @gba
     Abstract Scenario: kind volledig onbekend, veld met onbekend waarde wordt gevraagd met fields
-      Gegeven de persoon met burgerservicenummer 999996150 heeft de volgende kinderen in de registratie
-      | Categorie | Burgerservicenummer (01.20) | Voornamen (02.10) | Voorvoegsel (02.30) | Geslachtsnaam (02.40) | Geboortedatum (03.10) | Gemeente document (82.10) | Datum document (82.20) | Beschrijving document (82.30) | Ingangsdatum geldigheid (85.10) | Datum van opneming (86.10) |
-      | 9         |                             |                   |                     | <geslachtsnaam>       | <geboortedatum>       | 1926                      | 20040105               | D27894-2004-A782              | 20031107                        | 20040112                   |
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999996150 |
+      En de persoon heeft een 'kind' met de volgende 'naam' gegevens
+      | naam                  | waarde          |
+      | geslachtsnaam (02.40) | <geslachtsnaam> |
+      En het 'kind' heeft de volgende 'geboorte' gegevens
+      | naam                  | waarde          |
+      | geboortedatum (03.10) | <geboortedatum> |
+      En het 'kind' heeft de volgende 'document' gegevens
```

Comment:
Waarom zijn de regels 106 t/m 110 opgenomen. Volgens mij hebben die geen invloed op dit scenario.

Reply:
wanneer er iets is van de gegevens van het kind, zoals geslachtsnaam of geboortedatum (ook al is het een standaardwaarde), bestaat het kind en wordt het opgenomen. Document enz. is dan niet meer relevant

wanneer alleen document, akte, opneming en/of geldigheid zijn opgenomen en verder niks, bestaat het kind niet (meer) en moet het niet worden geleverd. Dan is in de Gegeven opnemen dat document, akte, opneming en/of geldigheid een waarde hebben wel relevant. is hier al een scenario voor?